2021-01-15 20:00:35,124 INFO    MainThread:1512658 [wandb_setup.py:_flush():70] setting env: {}
2021-01-15 20:00:35,124 INFO    MainThread:1512658 [wandb_setup.py:_flush():70] multiprocessing start_methods=fork,spawn,forkserver
2021-01-15 20:00:35,124 INFO    MainThread:1512658 [wandb_setup.py:_flush():70] setting login settings: {}
2021-01-15 20:00:35,124 INFO    MainThread:1512658 [wandb_init.py:_log_setup():318] Logging user logs to /home/whiteashes/Project/FedML/fedml_experiments/standalone/fedavg/wandb/run-20210115_200035-36pua0sm/logs/debug.log
2021-01-15 20:00:35,124 INFO    MainThread:1512658 [wandb_init.py:_log_setup():319] Logging internal logs to /home/whiteashes/Project/FedML/fedml_experiments/standalone/fedavg/wandb/run-20210115_200035-36pua0sm/logs/debug-internal.log
2021-01-15 20:00:36,847 INFO    MainThread:1512658 [wandb_run.py:_console_start():1308] atexit reg
2021-01-15 20:00:36,847 INFO    MainThread:1512658 [wandb_run.py:_redirect():1175] redirect: SettingsConsole.REDIRECT
2021-01-15 20:00:36,847 INFO    MainThread:1512658 [wandb_run.py:_redirect():1178] Redirecting console.
2021-01-15 20:00:36,848 INFO    MainThread:1512658 [redirect.py:install():205] install start
2021-01-15 20:00:36,848 INFO    MainThread:1512658 [redirect.py:install():220] install stop
2021-01-15 20:00:36,848 INFO    MainThread:1512658 [redirect.py:install():205] install start
2021-01-15 20:00:36,848 INFO    MainThread:1512658 [redirect.py:install():220] install stop
2021-01-15 20:00:36,848 INFO    MainThread:1512658 [wandb_run.py:_redirect():1222] Redirects installed.
2021-01-15 20:02:38,767 INFO    MainThread:1512658 [wandb_run.py:_atexit_cleanup():1278] got exitcode: 1
2021-01-15 20:02:38,767 INFO    MainThread:1512658 [wandb_run.py:_restore():1250] restore
2021-01-15 20:02:38,767 INFO    MainThread:1512658 [redirect.py:uninstall():224] uninstall start
2021-01-15 20:02:38,767 INFO    MainThread:1512658 [redirect.py:_stop():279] _stop: stdout
2021-01-15 20:02:38,768 INFO    MainThread:1512658 [redirect.py:_stop():285] _stop closed: stdout
2021-01-15 20:02:38,768 INFO    stdout    :1512658 [redirect.py:_pipe_relay():121] relay done saw last write: stdout
2021-01-15 20:02:38,768 INFO    stdout    :1512658 [redirect.py:_pipe_relay():137] relay done done: stdout
2021-01-15 20:02:38,768 INFO    MainThread:1512658 [redirect.py:_stop():291] _stop joined: stdout
2021-01-15 20:02:38,768 INFO    MainThread:1512658 [redirect.py:_stop():293] _stop rd closed: stdout
2021-01-15 20:02:38,768 INFO    MainThread:1512658 [redirect.py:uninstall():228] uninstall done
2021-01-15 20:02:38,768 INFO    MainThread:1512658 [redirect.py:uninstall():224] uninstall start
2021-01-15 20:02:38,768 INFO    MainThread:1512658 [redirect.py:_stop():279] _stop: stderr
2021-01-15 20:02:38,768 INFO    MainThread:1512658 [redirect.py:_stop():285] _stop closed: stderr
2021-01-15 20:02:38,768 INFO    stderr    :1512658 [redirect.py:_pipe_relay():121] relay done saw last write: stderr
2021-01-15 20:02:38,768 INFO    stderr    :1512658 [redirect.py:_pipe_relay():137] relay done done: stderr
2021-01-15 20:02:38,768 INFO    MainThread:1512658 [redirect.py:_stop():291] _stop joined: stderr
2021-01-15 20:02:38,768 INFO    MainThread:1512658 [redirect.py:_stop():293] _stop rd closed: stderr
2021-01-15 20:02:38,768 INFO    MainThread:1512658 [redirect.py:uninstall():228] uninstall done
2021-01-15 20:02:39,246 INFO    MainThread:1512658 [wandb_run.py:_wait_for_finish():1395] got exit ret: uuid: "0e5f99b8617244aba56e1fae35000ae4"
response {
  poll_exit_response {
    file_counts {
      wandb_count: 1
    }
    pusher_stats {
      uploaded_bytes: 1160
      total_bytes: 1160
    }
  }
}

2021-01-15 20:02:41,260 INFO    MainThread:1512658 [wandb_run.py:_wait_for_finish():1395] got exit ret: uuid: "fae1973af21b4f9eaf8968f477fd4d0f"
response {
  poll_exit_response {
    done: true
    exit_result {
    }
    file_counts {
      wandb_count: 5
    }
    pusher_stats {
      uploaded_bytes: 4620
      total_bytes: 4620
    }
  }
}

2021-01-15 20:02:42,308 INFO    MainThread:1512658 [wandb_run.py:_show_files():1614] logging synced files
2021-01-15 20:02:42,309 INFO    MainThread:1512658 [internal.py:handle_exit():139] Internal process exited
